scripts/purge-info.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+echo 'Warning: this action will completely remove the database '${HOCS_DB_NAME}' and all its contents. It cannot be undone!'
+
+read -p 'Are you sure you want to remove the database (y/n)? ' yes_no
```

Comment:
I'm really not bothered as it's internal but shouldn't it be... `Y/n`?

Reply:
Really, it should be `y/N` because no is the default option.